src/compositor/seat/seat.c

```diff
@@ -114,8 +114,10 @@ seat_handle_key(struct wlc_seat *seat, const struct wlc_input_event *ev)
       return;
 
    wlc_keyboard_update_modifiers(&seat->keyboard, ev->device);
+   struct wlc_modifiers mods = {0, 0};
+   uint32_t keysym = wlc_keyboard_get_keysym_for_key_ptr(&seat->keyboard, ev->key.code, &mods);
 
-   if (seat->keyboard.modifiers.mods == (WLC_BIT_MOD_CTRL | WLC_BIT_MOD_ALT) && ev->key.code >= 59 && ev->key.code <= 88) {
+   if (seat->keyboard.modifiers.mods == (WLC_BIT_MOD_CTRL | WLC_BIT_MOD_ALT) && keysym >= XKB_KEY_F1 && keysym <= XKB_KEY_F12) {
```

Comment:
As long as the keysym constants increase linearly from F1 -> F12 (which I hope they do) This should be fine.